### PR TITLE
(do not merge) DOCS-5036 policy asset type

### PR DESCRIPTION
**Pull request by wwarner-salesforce**

### CODE REVIEW

**Comment by wwarner-salesforce** [comment]
questions:
- ~~Definition of policies: "Configuration modules for managing APIs."?~~
- ~~"Create an Asset" discusses ways to create each asset type. How do users create a new policy asset? Do they create the asset directly using the Exchange "New asset" menu?~~
- ~~"Create an Asset" shows an illustration of how each asset type appears in Exchange. How do policies fit into that picture?~~
- ~~"Download an Asset" explains downloading. For "As Mule policy" what does the jar file contain? For "As Policy definition" YAML is a possible result; is JSON also possible? What are the differences between the YAML file and the JAR file? Was the YAML file the original source in all cases?~~

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -39,6 +39,7 @@ Both the public and private content in Exchange can consist of the following kin
 * Connectors - Packaged connectivity to an endpoint developed and deployed on Anypoint Platform with third-party APIs and standard integration protocols. Use connectors within your application's flows to send and receive data using a protocol or specific API. Anypoint Studio comes with many bundled connectors, and Exchange has many more.
 * Templates - Packaged integration patterns built on best practices to address common use cases. You can add your information such as user credentials to complete the template's use case or solution, and you can customize or extend templates as needed.
 * Examples - Applications that are ready to run in Anypoint Studio and demonstrate a use case or solution.
+* Policies - Configuration modules for managing APIs.
```
Comment (Igua95) [inline modules/ROOT/pages/index.adoc]: Policies - Configuration modules that extend the functionality of an API and enforces certain capabilities such as security.

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 2]
```diff
@@ -4,12 +4,12 @@ include::_attributes.adoc[]
...
 :imagesdir: ../assets/images/
 
-In Anypoint Exchange private and Exchange portal, to share an API, example, template, or connector means that
+In Anypoint Exchange private and Exchange portal, to share an API, policy, example, template, or connector means that
 you first create an asset corresponding to the asset type.
 
 How you create an asset depends on its type.
 
-* For OAS 2.0, RAML, RAML fragments, HTTP, WSDL, and Custom assets, create the asset directly using the Exchange *New asset* menu.
+* For OAS 2.0, RAML, RAML fragments, HTTP, WSDL, Policy, and Custom assets, create the asset directly using the Exchange *New asset* menu.
```
Comment (Igua95) [inline modules/ROOT/pages/to-create-an-asset.adoc]: Please remove this, policies can only be created by Exchange Maven Facade. Should we clarify this somewhere?

Comment (Igua95) [reply modules/ROOT/pages/to-create-an-asset.adoc]: It's here https://docs.mulesoft.com/exchange/to-publish-assets-maven maybe it's enough. We can add the link, as the connectors: https://github.com/mulesoft/docs-exchange/pull/64/files#diff-4b7bcbde16b715290fbc84da32a3f1e2L14

File: modules/ROOT/pages/to-download-an-asset.adoc [hunk 3]
```diff
@@ -28,5 +28,9 @@ If you choose As RAML, Exchange downloads the API specification as a zip file co
 * Connector: Downloads as a jar file.
 * Template: Downloads as a jar or zip file.
 * Example: Downloads as a zip file.
+* Policy: Click the down arrow.
++
+If you choose As Mule policy, Exchange downloads the policy definition as a jar file containing a YAML file for the definition. If you choose As Policy definition, Exchange downloads the definition as a YAML file.
```
Comment (Igua95) [inline modules/ROOT/pages/to-download-an-asset.adoc]: If you choose As Mule policy, Exchange downloads the policy as a jar file containing. If you choose As Policy definition, Exchange downloads the policy definition as a YAML file.

**Review by Igua95 — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/index.adoc]: Much better. Fixed. Thank you!

File: modules/ROOT/pages/to-create-an-asset.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/to-create-an-asset.adoc]: Thank you! Removed policy creation from the *New asset* menu list and added it to the Maven list.

File: modules/ROOT/pages/to-download-an-asset.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/to-download-an-asset.adoc]: What does the jar file contain?

Comment (wwarner-salesforce) [reply modules/ROOT/pages/to-download-an-asset.adoc]: Federico explained on Slack that the jar is an executable java package that contains the policy itself, as part of the code that runs, which is always built 1:1 from the source YAML file. (Never JSON.)

**Comment by wwarner-salesforce** [comment]
OK, I believe this PR is complete and correct, and I have no further questions. @Igua95 please approve if it looks good to you.

**Review by Igua95 — APPROVED** [review]

**Review by kuntald — APPROVED** [review]
Looks good

**Review by kuntald — APPROVED** [review]
Looks good

File: modules/ROOT/pages/to-download-an-asset.adoc [hunk 4]
```diff
@@ -28,5 +28,9 @@ If you choose As RAML, Exchange downloads the API specification as a zip file co
 * Connector: Downloads as a jar file.
 * Template: Downloads as a jar or zip file.
 * Example: Downloads as a zip file.
+* Policy: Click the down arrow.
++
+If you choose *As Mule policy*, Exchange downloads the policy as a jar file. If you choose *As Policy definition*, Exchange downloads the definition YAML file.
```
Comment (dukesphere) [inline modules/ROOT/pages/to-download-an-asset.adoc]: I believe it's "JAR file" (all caps)

File: modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -39,6 +39,7 @@ Both the public and private content in Exchange can consist of the following kin
 * Connectors - Packaged connectivity to an endpoint developed and deployed on Anypoint Platform with third-party APIs and standard integration protocols. Use connectors within your application's flows to send and receive data using a protocol or specific API. Anypoint Studio comes with many bundled connectors, and Exchange has many more.
 * Templates - Packaged integration patterns built on best practices to address common use cases. You can add your information such as user credentials to complete the template's use case or solution, and you can customize or extend templates as needed.
 * Examples - Applications that are ready to run in Anypoint Studio and demonstrate a use case or solution.
+* Policies - Configuration modules to extend the functionality of an API and enforce capabilities such as security.
```
Comment (dukesphere) [inline modules/ROOT/pages/index.adoc]: comma before "such as" needed here?  -> ", such as" ?
https://www.grammarly.com/blog/such-as-comma/

**Review by dukesphere — APPROVED** [review]
approving with minor comments

File: modules/ROOT/pages/index.adoc (hunk 5 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/index.adoc]: Yes, that's better. Fixed.

File: modules/ROOT/pages/to-download-an-asset.adoc (hunk 4 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/to-download-an-asset.adoc]: Thank you! Fixed.

**Pull request merged**